code/clement/django/lab02/usersystem_app/templates/users/profile.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'base.html'%}
+
+{% block content %}
+
+    {% for blogpost in request.user.blogposts.all %}
+
+        <p>
+            
+            {{blogpost}}
+            
+        </p>
+        
+    {% endfor  %}
+
+{% endblock content %}
+
```

Comment:
I would love to see some more styling here. Using `{% empty %}` within a jinja for loop will allow us to render some content if the list is empty.

```
{% for item in items %}
    {{ item }}
{% empty %}
    No items!
{% endfor %}
```